coreconfig/config.go

```diff
@@ -73,13 +73,13 @@ var (
 )
 
 type systemConfig struct {
-	Autopilot *bool           `yaml:"autopilot,omitempty"`
-	Timezone  *string         `yaml:"timezone,omitempty"`
-	Hostname  *string         `yaml:"hostname,omitempty"`
-	Modprobe  *string         `yaml:"modprobe,omitempty"`
-	Modules   []string        `yaml:"load-kernel-modules,omitempty"`
-	Network   *networkConfig  `yaml:"network,omitempty"`
-	Watchdog  *watchdogConfig `yaml:"watchdog,omitempty"`
+	AutoUpdate *bool           `yaml:"autoupdate,omitempty"`
```

Comment:
backwards compatibility could be preserved here, but I'm pretty sure it's not worth it, given that (as it turns out) we hadn't actually documented this usage.